packages/noya-state/src/selectors.ts

```diff
@@ -134,6 +134,20 @@ export const getSelectedStyles = (state: ApplicationState): Sketch.Style[] => {
     : getSelectedLayerStyles(state).map((style) => style.value);
 };
 
+export const getSelectedTextStyles = (
```

Comment:
This can call `getSelectedTextLayers` rather than filtering within this function